editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/index.tsx

```diff
@@ -26,11 +37,14 @@ const GeneralSettings: React.FC = () => {
   useEffect(() => {
     const fetchTeam = async () => {
       try {
-        const fetchedTeam = await useStore.getState().fetchCurrentTeam();
-        if (!fetchedTeam) throw Error("Unable to find team");
+        const fetchedTeamSettings = await useStore
+          .getState()
+          .fetchCurrentTeamSettings();
+        console.log(fetchedTeamSettings);
+        if (!fetchedTeamSettings.settings) throw Error("Unable to find team");
 
         setFormikConfig({
-          initialValues: fetchedTeam.settings,
+          initialValues: fetchedTeamSettings.settings,
```

Comment:
Please see the other PR - it's idiomatically a little confusing / difficult to have `settings.settings`